tools/validate-inventory.py

```
@@ -0,0 +1,178 @@
+
+def main(args):
+    effective_vars = {}
+    ansible_success = False
+
+    # 1. Start by running ansible-inventory
+    try:
+        import subprocess
+        ansible_bin = shutil.which("ansible-inventory")
+        if ansible_bin:
+            cmd = [ansible_bin, "-i", "inventory/hosts.prod.yml", "--list"]
+            res = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)